DS-2440: Upgrade Angular output target version to support Angular v.19, Fix Angular POC errors + upgrade version to Angular 21 (#169)
ongov/ontario-design-system: packages/ontario-design-system-component-library-angular/README.md

## Introduction

> This library is built using [`@stencil/angular-output-target`](https://www.npmjs.com/package/@stencil/angular-output-target) and supports Angular versions 18–21. It is based off the [Ontario Design System Component Library](https://www.npmjs.com/package/@ongov/ontario-design-system-component-library) built using [Stencil](https://stenciljs.com/). For more information, [find it on NPM](https://www.npmjs.com/package/@ongov/ontario-design-system-component-library-angular).

Reviewer: We can't support any Angular versions less than `20` due to the `pkg:` updates that came out in that version.

```suggestion
This library is built using [`@stencil/angular-output-target`](https://www.npmjs.com/package/@stencil/angular-output-target) and supports Angular versions 20+. It is based off the [Ontario Design System Component Library](https://www.npmjs.com/package/@ongov/ontario-design-system-component-library) built using [Stencil](https://stenciljs.com/). For more information, [find it on NPM](https://www.npmjs.com/package/@ongov/ontario-design-system-component-library-angular).
```

## Installation and usage

To find documentation on individual web components in this component library, please download and refer to our [component documentation](https://designsystem.ontario.ca/docs/documentation/for-developers/web-components.html#component-documentation).

To use the Ontario Design System Angular component library, follow these steps:

### 1. Install the npm package

```bash
npm install --save @ongov/ontario-design-system-component-library-angular
```

### 2. Importing the component library styles

There are three supported ways to include the Ontario Design System theme in your Angular application.

**Option A: Direct Import in Angular Styles**

      Add the following to your global `styles.scss` file:

      ```scss

>       @import 'pkg:@ongov/ontario-design-system-component-library-angular/styles/theme.scss';

Reviewer: We should avoid using or promoting `@import`, this is a depreciated pattern and has been replaced by `@use` and `@forward`.

```suggestion
      @forward 'pkg:@ongov/ontario-design-system-component-library-angular/styles/theme.scss';
```

      ```

**Option B: Create a Separate Styles File**

Alternatively, you can create a new SCSS file (e.g., `theme.scss`). If you need to override the global styles theme with a custom asset base path, you can create a local theme wrapper that forwards the global styles and sets `$asset-base-path`, then import that wrapper in your app entry point.

```scss
@forward 'pkg:@ongov/ontario-design-system-component-library-angular/styles/theme.scss' with (
	$asset-base-path: '/assets'
);
```

Then, include this file in your Angular project by adding it to the `styles` array in `angular.json`:

```json
"styles": [
   "src/theme.scss"
]
```

**Option C: Add the Design System theme CSS**

> Add the following to the styles array in your project’s angular.json file:

Reviewer: ```suggestion
Add the following to the styles array in your project’s `angular.json` file:
```
